### PR TITLE
Add response links support

### PR DESCRIPTION
**Pull request by guyarb**

### CODE REVIEW

File: openapi3/paths.py [hunk 1]
```diff
@@ -386,6 +386,29 @@ def _parse_data(self):
...
-        raw_links        = self._get('links', dict)
+        self.links       = self._get('links', ['Link'], is_map=True)
 
-        # TODO - raw_headers and raw_links
+
+class Link(ObjectBase):
+    """
+    A `Link Object`_ describes a single Link from an API Operation Response to an API operation Request
+
+    .. _Response Object: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#linkObject
```
Comment (barweiss) [inline openapi3/paths.py:396]:
```suggestion
    .. _LinkObject: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.1.md#linkObject
```

File: openapi3/paths.py [hunk 2]
```diff
@@ -386,6 +386,29 @@ def _parse_data(self):
...
         raw_content      = self._get('content', dict)
         raw_headers      = self._get('headers', dict)
-        raw_links        = self._get('links', dict)
+        self.links       = self._get('links', ['Link'], is_map=True)
 
-        # TODO - raw_headers and raw_links
+
+class Link(ObjectBase):
+    """
+    A `Link Object`_ describes a single Link from an API Operation Response to an API operation Request
```
Comment (barweiss) [inline openapi3/paths.py:394]: Match case of "Operation"

**Review by barweiss — CHANGES_REQUESTED** [review]

**Review by barweiss — APPROVED** [review]

File: openapi3/paths.py (hunk 1 above)
Comment (guyarb) [reply openapi3/paths.py]: fixed

File: openapi3/paths.py (hunk 2 above)
Comment (guyarb) [reply openapi3/paths.py]: fixed

**Comment by guyarb** [comment]
@Dorthu

**Comment by Dorthu** [comment]
Sorry I missed this, takin' a look now

**Review by Dorthu — APPROVED** [review]
Looks good 👍   I'm going to follow this up with a few minor changes

Sorry it took so long for me to look at it, thanks for addition!

**Comment by Dorthu** [comment]
This was released in 1.5.0 this morning

**Pull request merged**